frontend/src/joi_schemas/group.js

```diff
@@ -0,0 +1,25 @@
+import Joi from 'joi';
+
+const schema = Joi.alternatives().try(
+  Joi.object().keys({
+    name: Joi.string().label('Name').required(),
+    slug: Joi.string().label('Slug').required(),
+    currency: Joi.string().label('Currency'),
+    website: Joi.string().uri().allow('').label('Website url'),
```

Comment:
**NOTICE**

If I remember right, `Joi.uri()` Will correctly (by the RFC standards) invalidate a url without a **protocol**: 

So while `http://google.com` is valid, `google.com` is not. 

Something to keep in mind.